evil-colemak-basics.el

```diff
@@ -146,6 +146,17 @@ rotated; see evil-colemak-basics-rotate-t-f-j."
         "m" 'evil-backward-char)
       (evil-define-key '(normal visual) keymap
         "h" 'evil-set-marker))
+    (when evil-respect-visual-line-mode
+      (evil-define-key '(motion normal visual-line-mode) keymap
```

Comment:
are you sure this is correct? isn't this supposed to be `visual`?

Reply:
That's how it's written in the source code of evil-mode: https://github.com/emacs-evil/evil/blob/master/evil-integration.el#L478

Reply:
I did some experimenting and it seems like `visual` works better.